### PR TITLE
Add court time

### PR DESCRIPTION
**Pull request by ctwoolsey**

### CODE REVIEW

File: src/main/java/svc/data/citations/CitationDAO.java [hunk 1]
```diff
@@ -93,15 +96,21 @@ public Citation mapRow(ResultSet rs, int i) {
 			try {	
 				citation.id = rs.getInt("id");
 				citation.citation_number = rs.getString("citation_number");
-				citation.citation_date = rs.getDate("citation_date");
+				citation.citation_date = (rs.getDate("citation_date")!=null)?rs.getDate("citation_date").toLocalDate():null;
```
Comment (abobwhite) [inline src/main/java/svc/data/citations/CitationDAO.java]: Seems inefficient to try to get the thing twice. Just get it in a local var and check. Also - "rs" (ResultSet) may have a "hasProperty" type of call.

File: src/main/java/svc/data/citations/CitationDAO.java [hunk 2]
```diff
@@ -93,15 +96,21 @@ public Citation mapRow(ResultSet rs, int i) {
...
 				citation.first_name = rs.getString("first_name");
 				citation.last_name = rs.getString("last_name");
-				citation.date_of_birth = rs.getDate("date_of_birth");
+				citation.date_of_birth = (rs.getDate("date_of_birth")!=null)?rs.getDate("date_of_birth").toLocalDate():null;
```
Comment (abobwhite) [inline src/main/java/svc/data/citations/CitationDAO.java]: Same here

File: src/main/java/svc/data/citations/CitationDAO.java [hunk 3]
```diff
@@ -93,15 +96,21 @@ public Citation mapRow(ResultSet rs, int i) {
...
 				citation.defendant_address = rs.getString("defendant_address");
 				citation.defendant_city = rs.getString("defendant_city");
 				citation.defendant_state = rs.getString("defendant_state");
 				citation.drivers_license_number = rs.getString("drivers_license_number");
-				citation.court_date = rs.getDate("court_date");
+				LocalDate courtDate = (rs.getDate("court_date")!=null)?rs.getDate("court_date").toLocalDate():null;
```
Comment (abobwhite) [inline src/main/java/svc/data/citations/CitationDAO.java]: And these

File: src/main/java/svc/data/citations/ViolationDAO.java [hunk 4]
```diff
@@ -70,7 +70,7 @@ public Violation mapRow(ResultSet rs, int i) throws SQLException
 				violation.warrant_status = rs.getBoolean("warrant_status");
 				violation.warrant_number = rs.getString("warrant_number");
 				violation.status = VIOLATION_STATUS.convertDatabaseStatusToEnum(rs.getString("status"));
-				violation.status_date = rs.getDate("status_date");
+				violation.status_date = (rs.getDate("status_date")!=null)?rs.getDate("status_date").toLocalDate():null;
```
Comment (abobwhite) [inline src/main/java/svc/data/citations/ViolationDAO.java]: And this

File: src/main/java/svc/data/citations/CitationDAO.java (hunk 1 above)
Comment (ctwoolsey) [inline src/main/java/svc/data/citations/CitationDAO.java]: Fixed

File: src/main/java/svc/data/citations/CitationDAO.java (hunk 2 above)
Comment (ctwoolsey) [inline src/main/java/svc/data/citations/CitationDAO.java]: Fixed

File: src/main/java/svc/data/citations/CitationDAO.java (hunk 3 above)
Comment (ctwoolsey) [inline src/main/java/svc/data/citations/CitationDAO.java]: Fixed

File: src/main/java/svc/data/citations/ViolationDAO.java (hunk 4 above)
Comment (ctwoolsey) [inline src/main/java/svc/data/citations/ViolationDAO.java]: Fixed

**Comment by abobwhite** [comment]
Approved but needs conflicts merged

**Comment by ctwoolsey** [comment]
Fixed Merge Conflict

**Pull request merged**